pymodbus/server/simulator/http_server.py

```diff
@@ -5,13 +5,13 @@
 import json
 import os
 from time import time
-from typing import List
+from typing import Any, List, Optional, cast
 
 
 try:
     from aiohttp import web
 except ImportError:
-    web = None
+    web = cast(Any, None)
```

Comment:
This seems very wrong.

Reply:
Yes. The variable `web` should not be used as package and as flag if the package is available.

Reply:
I am sorry that is a very traditional way to include optional packages.

If the package is not installed the variable "web" still need to be declared.